### PR TITLE
@types/node: Add missing 'verbatim' option in dns.resolve

### PR DESCRIPTION
**Pull request by murgatroid99**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/dns.html#dns_dns_lookup_hostname_options_callback
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by ZaneHannanAU — APPROVED** [review]

**Review by islishude — APPROVED** [review]

**Comment by murgatroid99** [comment]
OK, I'll do it when I'm back at work on Monday.

**Review by BrunoScheufler — APPROVED** [review]

File: types/node/index.d.ts [hunk 1]
```diff
@@ -2651,6 +2651,7 @@ declare module "dns" {
         family?: number;
         hints?: number;
         all?: boolean;
+        verbatim?: boolean;
```
Comment (SimonSchick) [inline types/node/index.d.ts]: Docs?

**Review by SimonSchick — APPROVED** [review]

File: types/node/index.d.ts (hunk 1 above)
Comment (murgatroid99) [reply types/node/index.d.ts]: I'm not sure what you're asking for. If you're referring to the documentation for this option, I linked to it in the original PR comment.

**Comment by murgatroid99** [comment]
I have added the changes for the Node 8 types as requested.

**Comment by r3nya** [comment]
@murgatroid99 thanks for your job!

**Review by r3nya — APPROVED** [review]

**Pull request merged**